lib/dolly/document.rb

```diff
@@ -51,7 +51,8 @@ def rev= value
       doc['_rev'] = value
     end
 
-    def save
+    def save options = {}
+      return false unless options[:validate] == false || valid?
```

Comment:
We are not implementing validations, that is not dolly concern, webare just giving the app an interface for implementing it. Dolly should work independently of rails